src/contexts/ConfigurationContext.tsx

```diff
@@ -406,10 +415,9 @@ export const ConfigContextProvider = ({ children }: Props) => {
         isEnabled: isTrue(config_rerank),
         numResults: config_rerank_num_results ?? rerank.numResults,
       });
-
     };
     loadConfig();
-  }, []);
+  }, [isConfigLoaded]);
```

Comment:
Won't this dependency cause this effect to fire twice? I think we want to keep this an empty array. With an empty array, this pattern essentially says "run this logic once, when the component is mounted."

Reply:
Yes